test/qa/test_firewall.py

```diff
@@ -57,6 +58,78 @@ def test_firewall_01():
     assert not firewall.is_active()
 
 
+@pytest.mark.flaky(reruns=2, reruns_delay=90)
+@timeout_decorator.timeout(40)
+def test_connected_firewall_enable():
+    lib.set_firewall("off")
+    assert not firewall.is_active()
+
+    output = sh.nordvpn.connect()
+
+    print(output)
+    assert lib.is_connect_successful(output)
+
+    with lib.ErrorDefer(sh.nordvpn.disconnect):
```

Comment:
I guess you just copied from the tests above, but should probably fix these defers just like you did in previous test suites.